python/cucim/src/cucim/skimage/_vendored/_ndimage_filters.py

```diff
@@ -231,7 +234,7 @@ def _correlate_or_convolve1d(
     default_algorithm = False
     if algorithm is None:
         default_algorithm = True
-        if input.ndim == 2 and weights.size <= 256:
+        if input.ndim == 2 and weights.size <= 256 and _is_not_windows:
```

Comment:
Would check for Windows before the other stuff. This can shortcut evaluation on Windows

```suggestion
        if _is_not_windows and input.ndim == 2 and weights.size <= 256:
```